owncloudData/src/main/java/com/owncloud/android/data/ClientManager.kt

```diff
@@ -34,6 +37,32 @@ class ClientManager(
     private val preferencesProvider: SharedPreferencesProvider,
     val context: Context
 ) {
+    // This client will maintain cookies across the whole login process.
+    private var ownCloudClient: OwnCloudClient? = null
+
+    /**
+     * Returns a client for the login process.
+     * Helpful to keep the cookies from the status request to the final login and user info retrieval.
+     * For regular uses, use [getClientForAccount]
+     */
+    fun getClientForUnExistingAccount(
```

Comment:
Un**e**xisting ?

Reply:
Do you mean that there is a typo? 👀

Reply:
First I was going to suggest `AnExisting` because I thought it was a typo, then I search unexisting in Google and since it's a single word I would write Unexisting instead of UnExisting... 😄 not very important anyway